### ISSUE TITLE
[CONF] Fetch system status

### ISSUE THREAD

**Issue by luka-j** [luka-j/matf-rs2-api-gateway#54]
Add an endpoint which will return details about API and RP microservice status: how many instances are there and when was the last time any config was updated at each of them. (depends on #52)
List of interesting properties will probably grow over time.

**Comment by luka-j** [comment]
This is a nice to have, but pretty low priority at the moment.
